main.py

```diff
@@ -76,6 +76,7 @@ def get_inps(
     """
     print("catching layer inputs from data", flush=True)
     layers = get_layers(model)
+
```

Comment:
Let's avoid changing lines that are not related to the PR. This messes up `git blame`. Thanks

Reply:
> Auto-formatting does that on its own sometimes.
It does. Still, It's better to fix it.

Reply:
Fixed ✅